app/models/health/qualifying_activity.rb

```diff
@@ -617,13 +617,74 @@ def occurred_within_three_months_of_enrollment?
       date_of_activity.present? && patient.first_n_contributed_days_of_enrollment(90).include?(date_of_activity)
     end
 
+    # at the time of this call does the patient have
+    # a valid care plan covering the date_of_activity
     def patient_has_valid_care_plan?
       return false if patient.care_plan_renewal_date.blank?
       return false unless date_of_activity.present?
 
       date_of_activity >= patient.care_plan_provider_signed_date && date_of_activity < patient.care_plan_renewal_date
     end
 
+
+    # For the date of ac date_of_activity did the patient have
+    # a care plan that MassHealth would consider
+    # valid covering the date_of_activity. This is much
+    # slower and more complex than patient_has_valid_care_plan?
+    # which cares only about the patients status now
+    def patient_had_valid_care_plan?
+      # These have changed over time
+      # FIXME?: do we want to back fill rules in effect
```

Comment:
We do not.  There was no "grandfathering"

Reply:
Great